system/CLI/CLI.php

```diff
@@ -272,7 +273,35 @@ public static function prompt(string $field, $options = null, string $validation
 			}
 		}
 
-		return empty($input) ? '' : $input;
+		return (string) $input;
+	}
+
+	/**
+	 * Asks the user to select a specific option out of a set of options based on the option's key
+	 *
+	 * @param string            $text       Output "field" text
+	 * @param array             $options    a list of options (array(key => description)), the first option will be the default value
+	 * @param string|array|null $validation Validation rules
+	 *
+	 * @return string The selected key
+	 *
+	 * @codeCoverageIgnore
+	 */
+	public static function promptSelect(string $text, array $options, $validation = null): string
+	{
+		if (!$options) {
+			throw new InvalidArgumentException('No options to select from provided');
+		}
+
+		CLI::write($text);
+
+		$keyMaxLength = max(array_map('mb_strlen', array_keys($options)));
```

Comment:
```suggestion
		$keyMaxLength = max(array_map('mb_strwidth', array_keys($options)));
```

Reply:
I would love to see an explanation of the difference :D

Reply:
We have a PR before for CLI::strlen in which the underlying is mb_strlen. However, that multibyte does not work as expected with japanese characters. So it was found mb_strwidth would work for that

Reply:
Alright thanks 👍